legend-sdlc-server/src/main/resources/docker/config/config.json

```diff
@@ -0,0 +1,106 @@
+{
+  "applicationName": "Legend SDLC",
+  "server": {
+    "adminContextPath": "/admin",
+    "applicationConnectors": [
+      {
+        "type": "http",
+        "port": ${SDLC_PORT},
+        "maxRequestHeaderSize": "128KiB"
+      }
+    ],
+    "adminConnectors": [
+      {
+        "type": "http",
+        "port": 7071
+      }
+    ],
+    "gzip": {
+      "includedMethods": [
+        "GET",
+        "POST"
+      ]
+    },
+    "requestLog": {
+      "type": "classic",
+      "level": "OFF",
+      "appenders": [
+        {
+          "type": "console",
+          "logFormat": "OFF"
+        }
+      ]
+    },
+    "rootPath": "/api"
+  },
+  "projectStructure": {
+    "projectCreation": {
+      "groupIdPattern": "^org\\.finos\\.legend\\..+"
+    },
+    "extensionProvider": {
+      "org.finos.legend.sdlc.server.gitlab.finos.FinosGitlabProjectStructureExtensionProvider": {}
+    }
+  },
+  "filterPriorities": {
+    "GitLab": 1,
+    "org.pac4j.j2e.filter.CallbackFilter": 2,
+    "org.pac4j.j2e.filter.SecurityFilter": 3,
+    "CORS": 4
+  },
+  "pac4j": {
+    "callbackPrefix": "/api/pac4j/login",
+    "mongoUri": "${MONGODB_URI}",
+    "mongoDb": "${MONGODB_NAME}",
```

Comment:
Are the target users going to have MongoDB available?

Reply:
Our instructions will ask them to spin one up. Also, we will supply a Docker compose manifest that spins one up.